draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -145,7 +145,13 @@ If a provider has an application-specific need to have multiple verifications fo
 
 This again allows the provider to query only for application-specific records it needs, while giving flexibility to the user adding the DNS verification record (i.e. they can be given permission to only add records under a specific prefix by the DNS administrator). Whether or not multiple verifying records can exist for the same domain is up to the implementation.
 
-Providers MUST provide clear instructions on when a verifying record can be removed. The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete.
+Providers MUST provide clear instructions on when a verifying record can be removed. The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete. These instructions MAY be encoded in the RDATA via comma-separated ASCII key-value pairs {{RFC1464}} using the key `expiry`. If this is done, the token should have a key `token`. For example:
+
+    _foo-challenge.example.com.  IN   TXT  "token=3419...3d206c4,expiry=2023-02-08T02:03:19+00:00"
+
+Alternatively, if the record should never expire, the `expiry` can set to be `never`.
+
+    _foo-challenge.example.com.  IN   TXT  "token=3419...3d206c4,expiry=never"
```

Comment:
and perhaps the word/prefix "challenge" should be changed to proof-of-ownership or "permission" or "service"? I think _foo-service.example.com would be the clearest ? or _foo-auth-service ?

Reply:
I kept it as "challenge" to align with what Let's Encrypt does.

Reply:
I don't think there is value in doing that?